examples/drag_and_drop/src/lib.rs

```diff
@@ -0,0 +1,198 @@
+use seed::{prelude::*, *};
+use web_sys::{self, DragEvent, Event, HtmlDivElement};
+// ------ ------
+//     Init
+// ------ ------
+
+fn init(_: Url, _: &mut impl Orders<Msg>) -> Model {
+    Model {
+        drop_zone: Default::default(),
+        drag_me_1: Default::default(),
+        drag_me_2: Default::default(),
```

Comment:
```suggestion
        drop_zone: ElRef::new(),
        drag_me_1: ElRef::new(),
        drag_me_2: ElRef::new(),
```
Or at least `ElRef::default()`. 
The reader has no idea what the type is with `Default::` on the first glance. (I assume Clippy will have the same opinion :))